AcsfToolsCommands.php

```diff
@@ -168,6 +168,12 @@ public function ml($cmd, $command_args = '', $command_options = '', $options = [
       $drush_command_options[$key] = $value;
     }
 
+    // Command always passes the default option as `yes` irrespective if `--no`
+    // option used. Pass confirmation as `no` if use that.
+    if ($options['no']) {
```

Comment:
@joshirohit100 Which code is actually passing 'yes' as default value?

Reply:
@vbouchet31 yet to find that.

Reply:
@vbouchet31 when we run acsf-tools-ml command without any y/n, confirmation code in drush command is like `$this->io()->confirm(dt('Do you want Yes/No?'))`

This calls `Drush\Style\DrushStyle::confirm()` and goes in condition -
```
if (Drush::affirmative()) {
            $this->comment($question . ': yes.');
            return true;
        }
```
and thus return true.

This `Drush::affirmative()` condition is like -
```
if (!static::hasService('input')) {
            throw new \Exception('No input service available.');
        }
        return Drush::input()->getOption('yes') || (Drush::backend() && !Drush::negative());
```
And here `Drush::backend() && !Drush::negative()` is true as we are in backend mode && there is no `--no` option available.

Reply:
@vbouchet31 just for a note again, `--no` and `-n` are not same options while `--yes` and `-y` are. So this won't work if we use `-n`. If we need this, we need the check for `no-interaction` option (as -n is short name for this).

Reply:
@joshirohit100 That is fine. I wanted to confirm the default confirmation was kind of on purpose. It is also fine that -n does not work as this no-interaction is a global option for all commands.